charts/kube-prometheus-stack/templates/prometheus-operator/clusterrole.yaml

```diff
@@ -1,8 +1,11 @@
 {{- if and .Values.prometheusOperator.enabled .Values.global.rbac.create }}
 apiVersion: rbac.authorization.k8s.io/v1
-kind: ClusterRole
+kind: {{ .Values.prometheusOperator.rbac.namespaced | ternary "Role" "ClusteRole" }}
```

Comment:
should `ClusteRole` be `ClusterRole` ?

Reply:
Thats yeah, currently just a brain dump PR